test/fail_compilation/ice15002.d

```diff
@@ -0,0 +1,10 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/ice15002.d(10): Error: array index 5 is out of bounds x[0 .. 3]
+fail_compilation/ice15002.d(10): Error: array index 5 is out of bounds x[0 .. 3]
```

Comment:
Note that, the duplicated error message is known front-end diagnostic problem, e.g. [issue 10910](https://issues.dlang.org/show_bug.cgi?id=10910).